### DISCUSSION TITLE
Offline Wikipedia

### DISCUSSION THREAD

**Discussion by ichnilatis-gr** [koreader/koreader#10460] [General]
I saw in issue #10304 that @jonnyl2 mentioned (offline) Wikipedia and I would like to ask if this is available anywhere.
Thanks!

**Comment by jonnyl2** [comment]
You can find it on the huzheng.org site under Babylon/Misc dictionaries. It is a few years old though and only has the summary articles. If there is anything more current and more comprehensive I'd also like to know.
